.gitignore

```diff
@@ -1,6 +1,6 @@
 
-# Created by https://www.gitignore.io/api/web,node,macos,linux,windows,visualstudiocode
-# Edit at https://www.gitignore.io/?templates=web,node,macos,linux,windows,visualstudiocode
+# Created by https://www.gitignore.io/api/node,macos,linux,windows,visualstudiocode
+# Edit at https://www.gitignore.io/?templates=node,macos,linux,windows,visualstudiocode
```

Comment:
Note that I removed web here because ignoring web did not do anything like I thought it would.